cpp/src/dbscan/dbscan_api.h

```diff
@@ -23,11 +23,13 @@ extern "C" {
 
 //Single precision version of DBSCAN fit
 cumlError_t cumlSpDbscanFit(cumlHandle_t handle, float* input,
```

Comment:
We should have header comments in this user-facing function. For example, it is not clear what max_bytes_per_batch means and what the verbosity levels are. The signature is specified in the wrapped C++ header.

Reply:
Done.